src/cata_utility.cpp

```diff
@@ -282,3 +282,8 @@ double convert_weight( int weight )
     }
     return ret;
 }
+
+double temp_to_celsius( float fahrenheit )
+{
+    return ( ( fahrenheit - 32 ) * 5 / 9 );
```

Comment:
Are you sure? I did recognize this as problematic as well, but `fahrenheit` is a float, and the calculation starts from it. The subtraction results in a float, the multiplications returns a float, and the final division as well.

The startling thing is: the input is a `float`, but the output is a `double`, why not make them both the same type?